core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.controller.spec.ts

```diff
@@ -145,7 +145,7 @@ describe('Customize Interaction Modal Controller', function() {
         ).toBe(7);
         expect(
           $scope.ALLOWED_INTERACTION_CATEGORIES[1].interaction_ids.length
-        ).toBe(7);
+        ).toBe(6);
```

Comment:
Why did this change?

Reply:
We are removing number with units from the android validation constant list, so it reduced to one.

Reply:
`android_validation_constants.py`

```
VALID_INTERACTION_IDS = [
    'AlgebraicExpressionInput', 'Continue', 'DragAndDropSortInput',
    'EndExploration', 'FractionInput', 'ImageClickInput', 'ItemSelectionInput',
    'MathEquationInput', 'MultipleChoiceInput', 'NumericExpressionInput',
    'NumericInput', 'RatioExpressionInput', 'TextInput'
]
```